src/components/screens/consul/Debates/DebateDetail.js

```diff
@@ -78,101 +77,100 @@ export const DebateDetail = ({ data, onRefresh, route, navigation }) => {
       onRefresh();
       setLoading(false);
       reset();
+      Keyboard.dismiss();
     } catch (err) {
       console.error(err);
     }
   };
 
   return (
-    <SafeAreaViewFlex>
-      <WrapperWithOrientation>
```

Comment:
i found in the commit message:

> - removed from detail pages in `ConsulDetailScreen`
>  because the main component is wrapped with
>  `WrapperWithOrientation` and `SafeAreaViewFlex`.

i was asking myself "what was removed"? i needed to take a look in the code to find out. could have been mentioned, that these components were removed, because already wrapped on a higher level.

Reply:
ok thanks.

did you understand what i mean? you wrote `removed from detail pages...`. 
but, what was removed?